cmd/logcounter/log_counter.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2018 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
+
+	"k8s.io/node-problem-detector/cmd/logcounter/options"
+	"k8s.io/node-problem-detector/pkg/logcounter"
+)
+
+const (
+	exitOK      = 0
+	exitNotOK   = 1
+	exitUnknown = 2
+)
+
+func main() {
+	fedo := options.NewLogCounterOptions()
+	fedo.AddFlags(pflag.CommandLine)
+	pflag.Parse()
+
+	counter, err := logcounter.NewKmsgLogCounter(fedo)
+	if err != nil {
+		glog.Error(err)
```

Comment:
Use `fmt.Print`.

We only support `stdout` now. https://github.com/kubernetes/node-problem-detector/blob/master/pkg/custompluginmonitor/plugin/plugin.go#L112

Reply:
should I use Printf, or should I pass --logtostderr to the binary?  I am leaning toward the second, but whatever you want works :p

Reply:
passed --logtostderr to the binary

Reply:
changed to printf